[2025-03-17T20:13:42.129+0800] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-17T20:13:42.143+0800] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: reddit_yf_scraper.upload_reddit_to_gcs scheduled__2025-03-16T00:00:00+00:00 [queued]>
[2025-03-17T20:13:42.152+0800] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: reddit_yf_scraper.upload_reddit_to_gcs scheduled__2025-03-16T00:00:00+00:00 [queued]>
[2025-03-17T20:13:42.152+0800] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-17T20:13:42.160+0800] {taskinstance.py:2890} INFO - Executing <Task(LocalFilesystemToGCSOperator): upload_reddit_to_gcs> on 2025-03-16 00:00:00+00:00
[2025-03-17T20:13:42.165+0800] {standard_task_runner.py:72} INFO - Started process 19637 to run task
[2025-03-17T20:13:42.173+0800] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'reddit_yf_scraper', 'upload_reddit_to_gcs', 'scheduled__2025-03-16T00:00:00+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/var/folders/xg/h9xj0trn3dxds251bj2q6hf80000gn/T/tmp5puuz4al']
[2025-03-17T20:13:42.175+0800] {standard_task_runner.py:105} INFO - Job 21: Subtask upload_reddit_to_gcs
[2025-03-17T20:13:42.211+0800] {task_command.py:467} INFO - Running <TaskInstance: reddit_yf_scraper.upload_reddit_to_gcs scheduled__2025-03-16T00:00:00+00:00 [running]> on host ervins-air-2
[2025-03-17T20:13:42.270+0800] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='reddit_yf_scraper' AIRFLOW_CTX_TASK_ID='upload_reddit_to_gcs' AIRFLOW_CTX_EXECUTION_DATE='2025-03-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-16T00:00:00+00:00'
[2025-03-17T20:13:42.271+0800] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-17T20:13:42.272+0800] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-17T20:13:42.272+0800] {logging_mixin.py:190} INFO - Current task name:upload_reddit_to_gcs state:running start_date:2025-03-17 12:13:42.144000+00:00
[2025-03-17T20:13:42.272+0800] {logging_mixin.py:190} INFO - Dag name:reddit_yf_scraper and current dag run status:running
[2025-03-17T20:13:42.272+0800] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-17T20:13:42.276+0800] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-03-17T20:13:42.278+0800] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2025-03-17T20:13:42.279+0800] {taskinstance.py:3313} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/Users/ervinyeoh/Desktop/unimods/is3107/project/.venv/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 768, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ervinyeoh/Desktop/unimods/is3107/project/.venv/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 734, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ervinyeoh/Desktop/unimods/is3107/project/.venv/lib/python3.11/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ervinyeoh/Desktop/unimods/is3107/project/.venv/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ervinyeoh/Desktop/unimods/is3107/project/.venv/lib/python3.11/site-packages/airflow/providers/google/cloud/transfers/local_to_gcs.py", line 102, in execute
    raise FileNotFoundError(self.src)
FileNotFoundError: /Users/ervinyeoh/Desktop/unimods/is3107/project/airflow/snp500-reddit/*.csv
[2025-03-17T20:13:42.286+0800] {logging_mixin.py:190} INFO - Task instance in failure state
[2025-03-17T20:13:42.286+0800] {logging_mixin.py:190} INFO - Task start:2025-03-17 12:13:42.144000+00:00 end:2025-03-17 12:13:42.285921+00:00 duration:0.141921
[2025-03-17T20:13:42.287+0800] {logging_mixin.py:190} INFO - Task:<Task(LocalFilesystemToGCSOperator): upload_reddit_to_gcs> dag:<DAG: reddit_yf_scraper> dagrun:<DagRun reddit_yf_scraper @ 2025-03-16 00:00:00+00:00: scheduled__2025-03-16T00:00:00+00:00, state:running, queued_at: 2025-03-17 12:12:24.992072+00:00. externally triggered: False>
[2025-03-17T20:13:42.287+0800] {logging_mixin.py:190} INFO - Failure caused by /Users/ervinyeoh/Desktop/unimods/is3107/project/airflow/snp500-reddit/*.csv
[2025-03-17T20:13:42.287+0800] {taskinstance.py:1226} INFO - Marking task as UP_FOR_RETRY. dag_id=reddit_yf_scraper, task_id=upload_reddit_to_gcs, run_id=scheduled__2025-03-16T00:00:00+00:00, execution_date=20250316T000000, start_date=20250317T121342, end_date=20250317T121342
[2025-03-17T20:13:42.298+0800] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-17T20:13:42.298+0800] {standard_task_runner.py:124} ERROR - Failed to execute job 21 for task upload_reddit_to_gcs (/Users/ervinyeoh/Desktop/unimods/is3107/project/airflow/snp500-reddit/*.csv; 19637)
Traceback (most recent call last):
  File "/Users/ervinyeoh/Desktop/unimods/is3107/project/.venv/lib/python3.11/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ervinyeoh/Desktop/unimods/is3107/project/.venv/lib/python3.11/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ervinyeoh/Desktop/unimods/is3107/project/.venv/lib/python3.11/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/Users/ervinyeoh/Desktop/unimods/is3107/project/.venv/lib/python3.11/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ervinyeoh/Desktop/unimods/is3107/project/.venv/lib/python3.11/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ervinyeoh/Desktop/unimods/is3107/project/.venv/lib/python3.11/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/Users/ervinyeoh/Desktop/unimods/is3107/project/.venv/lib/python3.11/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ervinyeoh/Desktop/unimods/is3107/project/.venv/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 3006, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/Users/ervinyeoh/Desktop/unimods/is3107/project/.venv/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 274, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/Users/ervinyeoh/Desktop/unimods/is3107/project/.venv/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 3161, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ervinyeoh/Desktop/unimods/is3107/project/.venv/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 3185, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ervinyeoh/Desktop/unimods/is3107/project/.venv/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 768, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ervinyeoh/Desktop/unimods/is3107/project/.venv/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 734, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ervinyeoh/Desktop/unimods/is3107/project/.venv/lib/python3.11/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ervinyeoh/Desktop/unimods/is3107/project/.venv/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ervinyeoh/Desktop/unimods/is3107/project/.venv/lib/python3.11/site-packages/airflow/providers/google/cloud/transfers/local_to_gcs.py", line 102, in execute
    raise FileNotFoundError(self.src)
FileNotFoundError: /Users/ervinyeoh/Desktop/unimods/is3107/project/airflow/snp500-reddit/*.csv
[2025-03-17T20:13:42.314+0800] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2025-03-17T20:13:42.327+0800] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-17T20:13:42.327+0800] {local_task_job_runner.py:245} INFO - ::endgroup::
